styx-api-service/src/main/java/com/spotify/styx/api/SchedulerProxyResource.java

```diff
@@ -48,12 +51,22 @@ public class SchedulerProxyResource {
 
   private final String schedulerServiceBaseUrl;
   private final Client client;
+  private final String schedulerHost;
 
   public SchedulerProxyResource(String schedulerServiceBaseUrl, Client client) {
     this.schedulerServiceBaseUrl = Objects.requireNonNull(schedulerServiceBaseUrl);
+    this.schedulerHost = getHost(schedulerServiceBaseUrl);
```

Comment:
```suggestion
    this.schedulerHost = URI.create(schedulerServiceBaseUrl).getHost();
```

Reply:
should I catch the exception in the class creating SchedulerProxyResource ?

Reply:
nvm missed your comment

Reply:
`URI.create` does not throw checked exception, and we can let it fail without handling.